db/scripts/find_cured.py

```diff
@@ -1,100 +1,105 @@
 
```

Comment:
The changes in this file are mainly reordering a few things and cleaning stuff up right? I just wanted to make sure there weren't any major functional changes I was missing.

Reply:
Yeah - I moved schema-related functions from `queries.py` to `schema.py` and those changes are reflected here